web-app/django/VIM/apps/instruments/static/instruments/js/DisplaySettings.js

```diff
@@ -67,25 +67,25 @@ function setMasonryView() {
     });
 }
 
-// Instrument badge settings
-const instrumentLanguage = document.querySelector("#instrument-language-element");
-const instrumentBadge = document.querySelector("#instrument-language-badge");
-
-updateInstrumentBadge();
-
-function updateInstrumentBadge() {
-    const hideInstrumentBadge = localStorage.getItem('hideInstrumentBadge') || false;
-    if (hideInstrumentBadge) {
-        instrumentBadge.style.display = "none";
-    } else {
-        instrumentBadge.style.display = "";
-    }
-}
-
-instrumentLanguage.addEventListener("click", function () {
-    localStorage.setItem("hideInstrumentBadge", true);
-    updateInstrumentBadge();
-});
+// // Instrument badge settings
```

Comment:
Is there a reason you comment instead of delete?

Reply:
These are Yinan's codes, and these changes are for #96 . If she thinks these changes are okay, I will delete them.